src/main/kotlin/blackjack/domain/Cards.kt

```diff
@@ -0,0 +1,50 @@
+package blackjack.domain
+
+class Cards(
+    private val cards: List<Card>
+) {
+
+    fun hitCard(card: Card) {
+        this.cards.toMutableList().add(card)
+    }
+
+    fun getOneCard(): Card {
+        return cards.first()
+    }
+
+    fun extractCardsInfoAsString(): String {
+        var cardsInfo = ""
+
+        cards.forEachIndexed { index, card ->
+            cardsInfo += "${card.rank.value}${card.symbol.symbolName}"
+            if (index != cards.lastIndex) {
+                cardsInfo += ", "
+            }
+        }
+        return cardsInfo
+    }
+
+    fun calculateCardsTotalValue(): Int {
+        var total = 0
+        cards.filter { it.rank.rank != ACE_RANK_MARK }.forEach { card ->
```

Comment:
Rank는 enum이니 만큼 직접 비교를 하면 어떨까요?

추가로 체이닝 되는 함수를 개행해주면 가독성이 더 좋아질 것 같아요.
```suggestion
    cards.filter { it.rank == ACE }
          .forEach { 
```

Reply:
직접 비교할 수 있도록 수정하였습니다! + 가독성을 위해 개행도 추가하였습니다!